packages/context/README.md

```diff
@@ -285,6 +287,94 @@ instanceChanges.subscribe('myObjectId', callback);
 instanceChanges.unsubscribe('myObjectId', callback);
 ```
 
+### Notification Connection
+
+-   [useNofitication](#usenotification)
+    -   [documentChanges](#documentchangessubscribeobjectidinstanceid-data-documentchange)
+    -   [instanceChanges](#instancechangessubscribeobjectid-instanceids-instancechange)
+
+#### `useNofitication()`
+
+Hook used to obtain an instanceChanges instance of `Notification` and a documentChanges instance of `Notification`.
+
+##### `documentChanges.subscribe(objectId, instanceId, (data: DocumentChange[]]) => {})`
+
+Subscribe to the specified object instance document changes.
+
+```javascript
+const { documentChanges } = useNotification();
+
+documentChanges.subscribe('myObjectId', 'myInstanceId', (data) => {
+    console.log(data);
+});
+```
+
+The data provided to the callback will be an array of `DocumentChange` which contains the
+following data:
+
+-   `objectId`
+    -   Object describing the instance associated with the updated document.
+-   `instanceId`
+    -   Instance that the updated document is associated with.
+-   `documentId`
+    -   Document that was updated.
+-   `type`
+    -   The type of update. Possible values are `BlobCreated`, `BlobDeleted`, and `BlobMetadataUpdated`.
+
+##### `documentChanges.unsubscribe(objectId, instanceId, (data: DocumentChange[]) => {})`
+
+Unsubscribe to the specified object instance document changes.
+
+Callback function is optional.
+If callback function is defined, you must pass the exact same Function instance as was previously passed to `documentChanges.subscribe`.
+Passing a different instance (even if the function body is the same) will not remove the subscription.
+
+```javascript
+const { documentChanges } = useNotification();
+
+const callback = (data: DocumentChange[]) => {
+    console.log(data);
+};
+
+documentChanges.subscribe('myObjectId' , 'myInstanceId', callback);
+
+documentChanges.unsubscribe('myObjectId', 'myInstanceId', callback);
+```
+
+##### `instanceChanges.subscribe(objectId, undefined, (instanceIds: InstanceChange[]) => {})`
```

Comment:
`instanceChanges` only needs `objectId`, there's no reason to have a second argument that's always ignored.

Also, just call the callback parameter something like `changes`.  They're not just instance ids.